src/pages/workspaces/workspace/Workflows.js

```diff
@@ -248,20 +308,21 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
 
   async exportMethod() {
     const { namespace, name } = this.props
-    const { selectedWorkflow } = this.state
+    const { selectedWorkflow, workflowRename, methodAjax, config } = this.state
 
     this.setState({ exporting: true })
 
     try {
-      const methodAjax = Ajax().Methods.method(selectedWorkflow.namespace, selectedWorkflow.name, selectedWorkflow.snapshotId)
-
-      const config = _.maxBy('snapshotId', await methodAjax.configs())
+      const newConfig = (config && workflowRename !== selectedWorkflow.name) ?
+        _.merge(config, { name: workflowRename, payloadObject: { name: workflowRename } }) : undefined
+      this.setState({ config: newConfig })
```

Comment:
There's no need to set the `config` state here. In fact, in case the ajax call fails, this might lead to a slightly incorrect state, since it would incorrectly persist the edited name.